.github/workflows/docs-review.yaml

```diff
@@ -14,8 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      - uses: tcort/github-action-markdown-link-check@a800ad5f1c35bf61987946fd31c15726a1c9f2ba # v1.1.0
```

Comment:
The version tag does not work? Because v1.1.0 is there:

https://github.com/tcort/github-action-markdown-link-check/tags

Reply:
Copied from CCM. Fixed.